buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java

```diff
@@ -1381,25 +1381,27 @@ private Path getExtractedDistributionDir() {
         return distributions.get(currentDistro).getExtracted().getSingleFile().toPath();
     }
 
-    private List<File> getInstalledFileSet(Action<? super PatternFilterable> filter) {
-        return Stream.concat(plugins.stream().map(Provider::get), modules.stream().map(Provider::get))
-            .filter(File::exists)
-            // TODO: We may be able to simplify this with Gradle 5.6
-            // https://docs.gradle.org/nightly/release-notes.html#improved-handling-of-zip-archives-on-classpaths
-            .map(zipFile -> archiveOperations.zipTree(zipFile).matching(filter))
-            .flatMap(tree -> tree.getFiles().stream())
-            .sorted(Comparator.comparing(File::getName))
+    private List<Provider<List<File>>> getInstalledFileSet(Action<? super PatternFilterable> filter) {
```

Comment:
@dblock this is the only change I am unsure about but it seems like the builds are rolling and nothing breaks, by chance do you know where these particular properties (`installedClasspath` & `installedFiles`) are user or referenced?

Reply:
🤷 I suppose we'll find out earlier than later